config/peerpods/podvm/aws-podvm-image-handler.sh

```diff
@@ -317,7 +317,7 @@ function create_ami() {
     prepare_source_code
 
     # Prepare the pause image for embedding into the ami
-    download_and_extract_pause_image "${PAUSE_IMAGE_REPO}" "${PAUSE_IMAGE_VERSION}" "${PAUSE_IMAGE_REPO_AUTH_FILE}"
+    download_and_extract_pause_image "${PAUSE_IMAGE_REPO}" "${PAUSE_IMAGE_VERSION}" "${CLUSTER_PULL_SECRET_AUTH_FILE}"
```

Comment:
/lgtm
I'm acking individual commits. Looks like some of these commits can be in a separate PR and merged sooner before the bootc one..